lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -3,8 +3,11 @@ module Adapter
     class Attributes < Base
       def initialize(serializer, options = {})
         super
-        @include_tree = ActiveModel::Serializer::IncludeTree.from_include_args(options[:include] || '*')
-        @cached_attributes = options[:cache_attributes] || {}
+        @include_directive = options.fetch(:include_directive) do
```

Comment:
This is a private/internal "option". It allows to pass an instance of `JSONAPI::IncludeDirective` directly, instead of a string that will be parsed later. It is only used to avoid unnecessary recomputation of the `IncludeDirective` upon recursive calls of the `Attributes` adapter.